documentation/database/sqlserver.md

```diff
@@ -139,22 +139,22 @@ Example: `jdbc:sqlserver://<i>host</i>:<i>port</i>;databaseName=<i>database</i>;
 
 ### Azure Active Directory
 
-#### Installing ADAL4J
+#### Installing MSAL4J
```

Comment:
is it worth mentioning the change to MSAL4J from ADAL4J otherwise existing users may gloss over this and miss the change

Reply:
I updated the docs, but didn't know how to word it or where to put it, so if you got suggestions, feel free to suggest them :D

Reply:
I don't think we need to do this, since we only ever show the latest docs